spec/integration/about_page_spec.rb

```diff
@@ -2,11 +2,7 @@
 
 describe 'about_page', :type => :request do
   before :each do
-    @current_user = double(:webauth_user, :login => 'sunetid', :logged_in? => true, :privgroup => ADMIN_GROUPS.first)
-    allow(@current_user).to receive(:is_admin).and_return(true)
-    allow(@current_user).to receive(:roles).and_return([])
-    allow(@current_user).to receive(:is_manager).and_return(false)
-    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@current_user)
+    admin_user # see spec_helper
```

Comment:
All of this before code is handled in the `admin_user` helper method. There is no point in defining is_manager, roles etc. because the is_admin value overrides all other privileges, which can be clearly seen in the User model.